packages/web/src/pages/pay-and-earn-page/Purchases.tsx

```diff
@@ -174,55 +159,50 @@ const RenderPurchasesPage = () => {
     window.URL.revokeObjectURL(blobUrl)
   }, [userId])
 
-  const header = (
-    <Header
-      primary={messages.headerText}
-      rightDecorator={
-        <Button
-          onClick={downloadCSV}
-          variant={ButtonType.SECONDARY}
-          size={ButtonSize.SMALL}
-          iconLeft={IconCloudDownload}
-          disabled={isLoading || isEmpty}
-        >
-          {messages.downloadCSV}
-        </Button>
-      }
-    />
-  )
-
-  return (
-    <Page
-      title={messages.pageTitle}
-      description={messages.pageDescription}
-      header={header}
-    >
-      <div className={styles.container}>
-        {isEmpty ? (
-          <NoPurchases />
-        ) : (
-          <PurchasesTable
-            key='purchases'
-            data={purchases}
-            loading={isLoading}
-            onSort={onSort}
-            onClickRow={onClickRow}
-            fetchMore={fetchMore}
-            isVirtualized={true}
-            scrollRef={mainContentRef}
-            totalRowCount={count}
-            fetchBatchSize={TRANSACTIONS_BATCH_SIZE}
-          />
-        )}
-      </div>
-    </Page>
-  )
+  return {
+    count,
+    data: purchases,
+    fetchMore,
+    onSort,
+    onClickRow,
+    isEmpty,
+    isLoading,
+    downloadCSV
+  }
 }
 
-export const PurchasesPage = () => {
-  const { isLoaded, isEnabled } = useFlag(FeatureFlags.USDC_PURCHASES)
+/**
+ * Fetches and renders a table of purchases for the currently logged in user
+ * */
+export const Purchases = ({
```

Comment:
nit: `PurchasesTab` maybe? something a little more descriptive... maybe even `PurchasesTable` even though it shadows the presentational component?

Reply:
i like `PurchasesTab`